README.md

```diff
@@ -344,3 +344,27 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
 * For the given account, delete a role binding created at application level using the roleID `ocm support delete applicationRoleBinding [accountID] [roleID]`
 * For the given account, delete a role binding created at organization level using the roleID `ocm support delete organizationRoleBinding [accountID] [orgID] [roleID]`
 * For the given account, delete a role binding created at subscription level using the roleID `ocm support delete subscriptionRoleBinding [accountID] [subscriptionID] [roleID]`
+
+
+### Patch
+
+The `patch` command patches the given resource.
+
+#### Patching subscriptions
+
+Use the `subscriptions` subcommand to patch a subscription by passing the ID, or provide filter value to search matching subscriptions and patch them. Pass the JSON body for the patch request in terminal using `echo '{<PATCH_BODY>}' | ` before the actual command. By default the dry run flag will be enabled. Pass `dry-run=false` flag to actually remove the resource.
+
+The following flags are available for `delete capability`:
+
+```
+--filter                     If non-empty, filters and patches the matching subscriptions.
+--dryRun                     If false, it will execute the patch command call in instead of a dry run.
+--maxRecords                 Ensures that the maximum number of resources on which the operation is performed does not exceed than maxRecords, when passed dryRun as false. Defaults to 100.
+-h, --help                   help for patch
+```
+
+##### Examples
+
+* Patch a subscription by its ID and change the status to 'Reserved' (dry run) `echo '{ "status": "Reserved" }' | ocm support patch subscriptions [subID]`
+* Patch a subscription and change the support level to Self-Support for subscriptions with 'Reserved' status (no dry run) `echo '{ "support_level": "Self-Support" }' | ocm support patch subs --filter "status='Reserved' --dryRun=false`
+* Patch all subscriptions of an organization and change the status to Archived (dry run and setting maxRecords above the number of records) `echo '{ "status": "Archived" }' | ocm support patch subs --filter "organization_id='[orgID]' --dryRun=false --maxRecords=1000`
```

Comment:
Patch all subscriptions of an organization and change the status to Archived (dry run and setting **maxRecords above the number of records**) - the highlighted text is not so clear. Can you improve it?

Reply:
Changed to "set the maxRecords limit more than the actual number of affecting records

Reply:
Same comment as above regarding "affecting" vs "affected"

Reply:
Changed to "no dry run and set maxRecords more than the actual number of affected records"